### ISSUE TITLE
C# DocFX documentation shows wrong namespace

### ISSUE THREAD

**Issue by abjerner** [umbraco/UmbracoDocs#2641]
It seems that for all types in the C# documentation (or at least the ones I've checked), the namespace is listed as `System.Dynamic.ExpandoObject`, which is wrong:

Eg. https://our.umbraco.com/apidocs/v8/csharp/api/Umbraco.Core.Services.IDataTypeService.html

![image](https://user-images.githubusercontent.com/3634580/89743519-2ea0c000-daa4-11ea-8bec-64a4ed8e136e.png)

**Comment by sofietoft** [comment]
Thanks for the report @abjerner ! Much appreciated.

Please see my comment here: https://github.com/umbraco/UmbracoDocs/issues/2642#issuecomment-671215914

**Comment by jonat123** [comment]
@abjerner It seems that there has not been a lot of activity in regards to this issue 😅

So i was wondering if this is still a problem?

**Comment by abjerner** [comment]
@jonat123 Yes, it's still an issue. Both for the V8 and V9 versions of the docs - eg. here for V9:

https://apidocs.umbraco.com/v9/csharp/api/Umbraco.Cms.Core.Constants.Composing.html

**Comment by ronaldbarendse** [comment]
Looks like the template syntax has changed, [we are using](https://github.com/umbraco/Umbraco-CMS/blob/fc5612682035781cf0fc84f4d0d55fb87fa3ed9d/src/ApiDocs/umbracotemplate/partials/class.tmpl.partial#L18):

```
<h6><strong>{{__global.namespace}}</strong>:{{namespace}}</h6>
```

But the [default DocFX template uses](https://github.com/dotnet/docfx/blob/6f697ddb63ceecde0a20d2d2b640981fc68c3603/src/docfx.website.themes/default/partials/class.header.tmpl.partial#L45):

```
<h6><strong>{{__global.namespace}}</strong>: {{{namespace.specName.0.value}}}</h6>
```

**Comment by ronaldbarendse** [comment]
This is the PR that removed some backwards compatibility code in DocFX and required the above change to make it work again: https://github.com/dotnet/docfx/pull/1660. So we either didn't update DocFX for a while or nobody ever noticed this 😉

**Comment by jonat123** [comment]
The V8 API Docs are now fixed, and the API docs for V9 will be fixed in Umbraco 9.4.0 😄

**Comment by jonat123** [comment]
I'll close this issue down as it is fixed in v8 and will be fixed  9.4.0 :-)
